clients/python/github.com/AWS/GLIDE-for-Redis.json

```
@@ -0,0 +1,4 @@
+{
+    "name": "GLIDE for Redis",
+    "description": "General Language Independent Driver for the Enterprise (GLIDE) for Redis, is a high throughput Redis client, sponsored by AWS."
```

```suggestion
    "description": "General Language Independent Driver for the Enterprise (GLIDE) for Redis is an advanced multi-language Redis client that is feature rich, highly performant, and built for reliability and operational stability. GLIDE for Redis is supported by AWS."
```
Seems like a bit more descriptive?